test/end-to-end/tests/SolicitorHWFinAWP_test.js

```diff
@@ -1,7 +1,7 @@
 const testConfig = require('../config');
 
 Feature('Help with Fee in AWP - TS Solicitor application');
-Scenario('Solicitor Help With Fee in AWP @regression-suite', async({ I }) => {
+Scenario('Solicitor Help With Fee in AWP @to-fix', async({ I }) => {
```

Comment:
is this tag newly added or just for temporary

Reply:
Temporary tag as Test support event needs update for this scenario to work, to be enabled after fix